Issue38

katyhuff opened the pull request:
This adds a few paragraphs on the Polimi folks as well as the Delft folks. This closes #48.

katyhuff committed:
adds revision response doc -- uses the exam class

katyhuff committed:
minor revisions to revision doc

katyhuff committed:
This removes the overstatement about unmatched fidelity

katyhuff committed:
adds comment to revision response doc

katyhuff committed:
better merge solution.

katyhuff committed:
adds all of these refs to the updated bib file. See Zotero>paper-dev>2017-lindsay-moltres

katyhuff committed:
incorporates review of Aufiero, Laureau, and Fiorina.

katyhuff committed:
only use myriad once

katyhuff committed:
adds zanetti

katyhuff committed:
updates acros.tex with recent acros.tex

katyhuff committed:
updates acros.tex with recent acros.tex

katyhuff committed:
adds new text to solution

andrewryh commented on Moltres.tex:
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
+\cite{zanetti_geometric_2015} successfully combines high and low geometric 
+fidelity for graphite-moderated \glspl{MSR}.
+
+More recently, Aufiero et.  al \cite{aufiero_development_2014} have begun to
```
Aufiero et.  al**_._**

andrewryh commented on Moltres.tex:
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
+\cite{de_zwaan_static_2007,van_der_linden_2012} based on the approach in
+\cite{kophazi_development_2009}. These models showed good agreement for
+multiple accident transients. Meanwhile, leveraging lessons learned from these 
+efforts has resulted in a multiscale approach from Zanetti et al. 
```
Zanetti et**.** al.

andrewryh commented on Moltres.tex:
```diff
@@ -105,38 +105,52 @@ \section{Introduction}
+OpenFOAM multiphysics toolkit\cite{weller_tensorial_1998}.  This approach
+benefits from pre-implemented turbulence models available in the OpenFOAM
+library and captures the full-core three-dimensional geometry of the reactor
+primary circuit.  OpenFOAM \gls{CFD} has additionally been shown by Laureau et
```
Laureau et**.** al.

andrewryh committed:
Fixed "et al.": now it is the same everywhere

For multiple authors right way to cite is: **et al.**
http://davidson.libguides.com/c.php?g=349327&p=2357431

andrewryh committed:
Fixed "et al." everywhere

andrewryh reviewed (approved):
I only changed "et al." 3 times, nice job!

katyhuff commented:
thanks @andrewryh !

katyhuff commented:
Given @andrewryh is ok with this, I'll merge it in so that we can keep plugging along.
